src/ImageSharp/Metadata/Profiles/IPTC/IptcValue.cs

```diff
@@ -101,7 +101,7 @@ public string Value
                 if (this.Strict && value.Length > maxLength)
                 {
                     string cappedValue = value[..maxLength];
-                    valueBytes = this.encoding.GetBytes(cappedValue);
+                    valueBytes = this.encoding!.GetBytes(cappedValue);
```

Comment:
I think we should avoid doing this. It looks like the `Encoding` backing field should be set to `Encoding.UTF8` by default.